pkg/targetallocator/serviceaccount_test.go

```diff
@@ -0,0 +1,60 @@
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package targetallocator_test
```

Comment:
```suggestion
package targetallocator
```

Can't this be in the `targetallocator` package? Would align it with the rest of the `_test` files and also wouldn't need the import of `. "github.com/open-telemetry/opentelemetry-operator/pkg/targetallocator"` on line 24

Reply:
ah yeah, i was following the testing convention in the `pkg/collector` will change :)